toot.gemspec

```diff
@@ -1,29 +1,37 @@
-# coding: utf-8
-lib = File.expand_path('../lib', __FILE__)
+# frozen_string_literal: true
+
+lib = File.expand_path('lib', __dir__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 require 'toot/version'
 
 Gem::Specification.new do |spec|
-  spec.name          = "toot"
+  spec.name          = 'toot'
   spec.version       = Toot::VERSION
-  spec.authors       = ["Travis Petticrew"]
-  spec.email         = ["travis@petticrew.net"]
+  spec.authors       = ['Travis Petticrew']
+  spec.email         = ['travis@petticrew.net']
 
-  spec.summary       = %q{Send and receive events from remote services over HTTP.}
-  spec.description   = %q{Send and receive events from remote services over HTTP.}
-  spec.homepage      = "https://github.com/watermarkchurch/toot"
+  spec.summary       = 'Send and receive events from remote services over HTTP.'
+  spec.description   = 'Send and receive events from remote services over HTTP.'
+  spec.homepage      = 'https://github.com/watermarkchurch/toot'
 
   spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
-  spec.bindir        = "exe"
+  spec.bindir        = 'exe'
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
-  spec.require_paths = ["lib"]
+  spec.require_paths = ['lib']
 
-  spec.add_dependency "sidekiq", ">=2"
-  spec.add_dependency "rack", ">=1"
-  spec.add_dependency "faraday", "<1"
+  spec.add_dependency 'activesupport'
```

Comment:
is this really a dependency?

Reply:
Yes because of `symbolize_keys`.  I could try to extract that out.